src/Game.java

```diff
@@ -31,6 +35,8 @@ public void playGame(){
 			else{
 				whoseTurn = player1;
 			}
+			//#4, never rechecks for a winner
+			winner();
```

Comment:
This is not needed.  The function is called next when it goes back up to the head of the loop.